packages/client/src/generation/TSClient/common.ts

```diff
@@ -331,8 +360,12 @@ type IsObject<T extends any> = T extends Array<any>
 ? False
 : T extends Date
 ? False
-: T extends Buffer
-? False
+${
```

Comment:
This can now be removed, we use `Uint8Array` instead of `Buffer`.

Reply:
Thanks! Updated